webapp-django/crashstats/crashstats/models.py

```diff
@@ -1085,19 +1086,14 @@ class CrontabberState(SocorroMiddleware):
     API_WHITELIST = None
 
 
-class BugzillaAPI(SocorroCommon):
-    base_url = settings.BZAPI_BASE_URL
-    username = password = None
-
-
-class BugzillaBugInfo(BugzillaAPI):
-
-    # This is for the whole model.
-    cache_seconds = 0
+class BugzillaBugInfo(SocorroCommon):
 
     # This is for how long we cache the metadata of each individual bug.
     BUG_CACHE_SECONDS = 60 * 60
 
+    # How patient we are with the Bugzilla REST API
+    BUGZILLA_REST_TIMEOUT = 5  # seconds
```

Comment:
I disagree. 5 seconds is a really long time. Sure, it can happen, but if it does it's probably just a temporary glitch. The new `requests_retry_session` machine takes care of it. Its default is 3 retries. At means, if it, the first time, takes more than 5 seconds (every time), it'll cancel that request and do another one after a 0.3 second sleep. The total time is going to be about 21.8 seconds. 
```
>>> 5 + 0.0 + 5 + 0.3 + 5 + 0.6 + 5 + 0.9
21.799999999999997
```
That means it gets a total of 22 seconds to sort itself out. 
And thanks to the way we do caching, all we're requesting is a few fields for just a few bug IDs. 

If we change 5s to 10s the whole thing will take 42 seconds before it gives up. 

22s should be plenty enough.